crafters/image/ImageReader/__init__.py

```diff
@@ -7,36 +7,50 @@
 
 class ImageReader(BaseCrafter):
     """
-    :class:`ImageReader` loads the image from the given file path and save the `ndarray` of the image in the Document.
+    :class:`ImageReader` loads the image from the given file path, bytes or ndarray, and emits the unified `ndarray` with
+    shape (h,w,depth).
     """
 
-    def __init__(self, channel_axis: int = -1, *args, **kwargs):
+    def __init__(self, color_channel_axis: int = -1, *args, **kwargs):
```

Comment:
Is it possible to keep the argument name as it was? A lot of examples depend on that and it would make our life easier

Reply:
sure, done